CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator+Camera.swift

```diff
@@ -133,22 +134,21 @@ extension GiniScreenAPICoordinator: CameraViewControllerDelegate {
         return false
     }
 
-    private func showOnboardingScreen(completion: @escaping () -> Void) {
-        cameraViewController?.hideCameraOverlay()
-        cameraViewController?.hideCaptureButton()
-
-        let vc = OnboardingViewController(nibName: "OnboardingViewController", bundle: giniCaptureBundle())
-            guard let cameraViewController = self.cameraViewController else { return }
-            cameraViewController.showCameraOverlay()
-            cameraViewController.showCaptureButton()
-                if giniConfiguration.fileImportSupportedTypes != GiniConfiguration.GiniCaptureImportFileTypes.none {
-                    // TODO: remove cameraViewController.showFileImportTip()
-                } else if giniConfiguration.qrCodeScanningEnabled {
-                    // TODO: removecameraViewController.showQrCodeTip()
-                }
-
-            completion()
-
+    private func showOnboardingScreen(
+        cameraViewController: CameraScreen,
+        completion: @escaping () -> Void) {
+        cameraViewController.hideCameraOverlay()
+        cameraViewController.hideCaptureButton()
+
+        let vc = OnboardingViewController.init()
```

Comment:
[Suggestion] Use the short version of the initializer. `OnboardingViewController()`